azure-pipelines.yml

```diff
@@ -96,7 +96,7 @@ jobs:
         yarn global add lerna
         yarn bootstrap
       name: Bootstrap
-    - script: echo Disabling e2e test til spectron has update for electron v6 # yarn test:e2e
```

Comment:
Errata: the commit message talked about electron 4.0.0. It should be 6.0.0 instead.